src/site-furniture/Header/_header.scss

```diff
@@ -77,15 +77,15 @@
     padding-top: 6px; // small adjustment to center the logo
 
     svg {
-      height: 24px;
+      height: 25.73px; // 25.73px is the height of the logo's container to make logo 20px
       width: 100%;
     }
 
     @include isHeaderDesktop {
       padding-top: 0;
 
       svg {
-        height: 28px;
+        height: 25.73px; // 25.73px is the height of the logo's container to make logo 20px
         width: 100%;
```

Comment:
Is this needed?

Reply:
the height of svg 
![image](https://github.com/user-attachments/assets/a2a4efc4-d36d-4229-a8c4-e398e0c387c2)

the logo itself
![image](https://github.com/user-attachments/assets/473cc47d-2aa1-4cb9-a89a-54e0db63171e)